### ISSUE TITLE
Uncaught TypeError: generator.gen is not a function

### ISSUE THREAD

**Issue by PDDStudio** [Bat41/valhalla#41]
[Enter steps to reproduce:]

1. `Right click on a folder` -> `Create new Class (Interface as well)`
2. Enter a Name
3. Error thrown.

**Atom**: 1.15.0 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.3
**Thrown From**: [valhalla](https://github.com/Bat41/valhalla) package 1.1.0

### Stack Trace

Uncaught TypeError: generator.gen is not a function

```
At /Users/pddstudio/.atom/packages/valhalla/lib/valhalla.js:321

TypeError: generator.gen is not a function
    at Object.onConfirm (/packages/valhalla/lib/valhalla.js:321:26)
    at Input.onConfirm (/packages/valhalla/lib/views/modals/input.js:48:18)
    at /packages/valhalla/lib/views/modals/input.js:21:89)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
    at /app.asar/src/command-registry.js:3:59
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:599:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:390:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/app.asar/src/window-event-handler.js:106:36)
    at /app.asar/src/window-event-handler.js:3:59)
```

### Commands

```
     -1:36.3.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-nord-atom-syntax.theme-nord-atom-ui)
     -0:08.2.0 valhalla:new-class (span.name.icon.icon-file-directory)
     -0:04.6.0 core:confirm (input.hidden-input)
```

### Non-Core Packages

```
api-workbench 0.8.43 
atom-ternjs 0.18.2 
autocomplete-modules 1.6.8 
builder 0.0.1 
busy 0.7.0 
clipboard-ide 0.7.0 
css-color-underline 1.0.1 
editorconfig 2.2.2 
es6-javascript 1.0.0 
format-shell 2.0.0 
javascript-snippets 1.2.1 
json-colorer 1.0.0 
language-babel 2.56.2 
language-unix-shell 0.40.1 
language-vala-modern 0.3.2 
linter 2.1.0 
linter-eslint 8.1.3 
linter-shellcheck 1.4.3 
minimap-pigments 0.2.2 
nord-atom-syntax 0.6.0 
nord-atom-ui 0.10.2 
nuclide 0.209.0 
pigments 0.39.0 
process-palette 0.12.0 
project-plus 0.9.0 
selly-syntax-ide 2.0.2 
selly-ui-ide 3.1.0 
TerminalColor 1.1.0 
tool-bar 1.1.0 
tool-bar-main 0.0.10 
valhalla 1.1.0 
```

**Comment by Bat41** [comment]
It should be fixed with the 1.1.2. 🙂

**Comment by PDDStudio** [comment]
@Bat41 as usual, thanks 😉
